CONTRIBUTING.md

```diff
@@ -276,5 +276,39 @@ Because we sometimes need to reuse some setup or assertions in our tests, we ten
 - `fun verifyMockMatchesState(mock, [args])`: methods verifying that a mock’s interaction. These methods must be of Unit type, and only call verifications with the Mockito framework.
 - `fun setupSomething()`: method to setup a complex test (should only be used in the Given part of a test).
 
+### Nightly Tests
 
+Each public API method in the SDK is covered by a test case in the `nightly-tests` module. All test cases are executed on a Bitrise emulator by a Datadog Synthetic Test every 24 hours. Each test case
+output is measured by 2 Datadog Monitors (one for performance and one for functionality). There are some best practices when writing a nightly test as follows: 
+
+- The test method name must follow the following format: [rootFeature]_[subFeature]_[method]_[additionalInfo] where:
+
+    1. rootFeature is one of the top level features (logs, rum, apm, et)
```

Comment:
probably it is better to apply code span to the `rootFeature` name and also to the cases below.